datafusion/sqllogictest/test_files/explain.slt

```diff
@@ -79,10 +79,7 @@ logical_plan
 01)Limit: skip=0, fetch=10
 02)--Sort: aggregate_test_100_with_order.c1 ASC NULLS LAST, fetch=10
 03)----TableScan: aggregate_test_100_with_order projection=[c1]
-physical_plan
-01)GlobalLimitExec: skip=0, fetch=10
-02)--CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/aggregate_test_100_order_by_c1_asc.csv]]}, projection=[c1], output_ordering=[c1@0 ASC NULLS LAST], has_header=true
-
+physical_plan CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/aggregate_test_100_order_by_c1_asc.csv]]}, projection=[c1], limit=10, output_ordering=[c1@0 ASC NULLS LAST], has_header=true
```

Comment:
The `GlobalLimitExec` is not needed here because the `CsvExec` already has a limit and there is a single partition ✅